### ISSUE TITLE
Increase Maximum Entities

### ISSUE THREAD

**Issue by cadika-orade** [JACoders/OpenJK#276]
Some popular custom maps simply don't work because they contain too many entities for the engine to handle. I'd like to suggest this hard limit be increased or in some way softened.

From what I understand, here are the benefits of the hard limit, and an argument against each:
1. Improve performance by limiting the number of entities. I can understand why the original devs included this limit a decade ago, but with modern systems boasting RAM and processor speeds several orders of magnitude about what JA was designed to fit on, I think the limit could be considered outdated.
2. Limit bandwidth usage. I can understand this for multiplayer play, but it hardly seems applicable for singleplayer. Even for multiplayer, I would consider it more the problem of the server admin to make sure they don't load a map with so many entities that it creates lag for everyone.
3. Improve performance predictability. Considering the vanilla maps (obviously) don't use more than the allowed number entities, any sort of instability would have to come from either custom maps with tons of entities or (as is typically my case) spawning hundreds of items and NPCs at once. Either way, I can't imagine anyone blaming performance issues on the OpenJK developers. As long as everything works for the vanilla maps, then you've done your job. Upping the limit would just allow some freedom for players who like to push things until they break.

It has also been mentioned that upping the entity limit would break mods. I'm no expert, but I fail to see how this could possibly break the vast majority of mods. It could possibly break hacked JA clients if the limit on the server were increased, or possibly break the client if it connects to modded vanilla server. Either way, it seems that could be avoided by not increasing the limit on the multiplayer code.

TL;DR: 90% of my JA play is spawning NPCs and staging battles, but the entity limit... limits that. Please fix it.
